validator/src/commands/run/args.rs

```diff
@@ -1664,12 +1664,12 @@ pub fn add_args<'a>(app: App<'a, 'a>, default_args: &'a DefaultArgs) -> App<'a,
             .help("EXPERIMENTAL: Enable XDP zero copy. Requires hardware support"),
     )
     .arg(
-        Arg::with_name("use_tpu_client_next")
-            .long("use-tpu-client-next")
+        Arg::with_name("use_connection_cache")
+            .long("use-connection-cache")
```

Comment:
Is the old arg present in some previous release ? We generally shouldn't be breaking old arguments

Reply:
> Is the old arg present in some previous release ?
No, the old flag was introduced in 2.3 and I would like to change it to the new one as part of 2.3. 

In terms of naming, do you suggest o rename `--enable-connection-cache` instead?

Reply:
Shoot, I thought I responded but I never hit send apparently 😢 

> No, the old flag was introduced in 2.3 and I would like to change it to the new one as part of 2.3.

Ok nice, no compatibility concerns so we can change on the fly

> In terms of naming, do you suggest o rename --enable-connection-cache instead?

Do you envision keeping the `connection-cache` option around long term, or will this go away when `tpu-client-next` is fully proven out ? If both are going to stick around, I might suggest `--tpu-send-transaction-mode` or something.

Also, I assume you have some good runtime with the new mode since you're making it the default ?

Reply:
I envision that ConnectionCache will be eclipsed in some future, but I would like to have it around in case if we want to switch back to test something.